cmd/libs/go2idl/generator/snippet_test.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
If you decide to rename snippet.go to snipper_writer.go, then also please rename this file.